src/App/Components/Profile/Profile.js

```diff
@@ -0,0 +1,392 @@
+import React, { Component } from 'react';
+import propTypes from 'prop-types';
+import ReactImageFallback from 'react-image-fallback';
+import SimpleReactValidator from 'simple-react-validator';
+import { connect } from 'react-redux';
+import numeral from 'numeral';
+import _ from 'underscore';
```

Comment:
Underscore is a deprecated version of lodash @alainburindi .
Import lodash instead please